src/main/kotlin/frc/robot/RobotContainer.kt

```diff
@@ -41,6 +41,7 @@ object RobotContainer {
 
     private val testController = CommandXboxController(2)
 
+    @AutoLogOutput
```

Comment:
I'm pretty sure this won't work. Did you test it? 
Check AdvantageKit docs for how this should be done.

Reply:
1dfd63e23ab4db630d9de910964fc84b876f2ce0